src/marinade.ts

```diff
@@ -10,6 +10,8 @@ import { MarinadeReferralGlobalState } from './marinade-referral-state/marinade-
 import { assertNotNullAndReturn } from './util/assert'
 import { TicketAccount } from './marinade-state/borsh/ticket-account'
 
+const STAKE_ACCOUNT_SIZE = 200
```

Comment:
`web3.StakeProgram.space` (maybe there is something missing in the middle of the path)